frontend/packages/ceph-storage-plugin/integration-tests/tests/add-capacity.scenario.ts

```diff
@@ -0,0 +1,129 @@
+import { execSync } from 'child_process';
+import * as _ from 'lodash';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { click } from '@console/shared/src/test-utils/utils';
+import { isNodeReady } from '@console/shared/src/selectors/node';
+import {
+  addCapacityLbl,
+  addCapacityBtn,
+  ocsOp,
+  storageClusterView,
+  storageClusterRow,
+} from '../views/add-capacity.view';
+
+const NAMESPACE = 'openshift-storage';
+const KIND = 'storagecluster';
+
+let clusterJSON;
+let previousCnt;
+let previousPods;
+let updatedClusterJSON;
+let updatedCnt;
+let updatedPods;
+
+const storageCluster = JSON.parse(
+  execSync(`kubectl get ${KIND} -n ${NAMESPACE} -o json`).toString(),
+);
+
+describe('Check availability of cluster', () => {
+  if (storageCluster.items[0]) {
+    it('Should check if the cluster is Ready for expansion', () => {
+      expect(_.get(storageCluster.items[0], 'status.phase')).toBe('Ready');
+    });
+  } else {
+    it('Should state that cluster is not ready for expansion', () => {
+      expect(storageCluster.items[0]).toBeUndefined();
+    });
+  }
+});
+
+if (storageCluster.items[0]) {
+  describe('Check add capacity functionality for ocs service', () => {
+    beforeAll(async () => {
+      // check whether the cluster is present or not, and check for its ready state
+      // if not present exit and console that no cluster is present, will be send in next PR
```

Comment:
@afreen23 You can remove this comment since you have added the checks now.

Reply:
ack